foreman_remote_execution_core.gemspec

```diff
@@ -17,6 +17,6 @@ DESC
   s.files = Dir['lib/foreman_remote_execution_core/**/*'] +
             ['lib/foreman_remote_execution_core.rb', 'LICENSE']
 
-  s.add_runtime_dependency('foreman-tasks-core', '~> 0.1.0')
+  s.add_runtime_dependency('foreman-tasks-core', '>= 0.1.3', '< 2')
```

Comment:
Nitpick: shouldn't this be `< 1`? Or perhaps we don't even need to keep the top restriction